### PR TITLE
Update README.adoc

### PR DESCRIPTION
**Pull request by Dawn-Davenport**

### CODE REVIEW

**Comment by gmaxey** [comment]
Was there a reason we dropped this text? 

Kaniko builds images inside a container or Kubernetes cluster.

File: README.adoc [hunk 1]
```diff
@@ -1,8 +1,6 @@
 = CloudBees action: Build and publish Docker images with Kaniko
 
-Use this action to build and publish Docker images using link:https://github.com/GoogleContainerTools/kaniko[Kaniko], which builds container images from a Dockerfile. Kaniko builds images inside a container or Kubernetes cluster.
-
-The action is also instrumented to report back to the workflow run the image and tags names for artifact traceability purposes.
+Use this action uses link:https://github.com/GoogleContainerTools/kaniko[Kaniko]to build images based upon a Dockerfile then publish the image to a Docker registry.  This action also reports the image and tag names to the workflow run for artifact traceability purposes.
```
Comment (Dawn-Davenport) [inline README.adoc:3]:
```suggestion
Use this action uses link:https://github.com/GoogleContainerTools/kaniko[Kaniko]to build images based upon a Dockerfile then publish the image to a Docker registry.  Kaniko builds images inside a container or kubernetes cluster. This action also reports the image and tag names to the workflow run for artifact traceability purposes.
```

**Review by kushicp — APPROVED** [review]

File: README.adoc [hunk 2]
```diff
@@ -1,8 +1,6 @@
 = CloudBees action: Build and publish Docker images with Kaniko
 
-Use this action to build and publish Docker images using link:https://github.com/GoogleContainerTools/kaniko[Kaniko], which builds container images from a Dockerfile. Kaniko builds images inside a container or Kubernetes cluster.
-
-The action is also instrumented to report back to the workflow run the image and tags names for artifact traceability purposes.
+Use this action uses link:https://github.com/GoogleContainerTools/kaniko[Kaniko]to build images based upon a Dockerfile then publish the image to a Docker registry.  Kaniko builds images inside a container or kubernetes cluster. This action also reports the image and tag names to the workflow run for artifact traceability purposes.
```
Comment (kushicp) [inline README.adoc:3]: The sentence isn't framed correctly here "_Use this action uses_"

Comment (Dawn-Davenport) [inline README.adoc:3]:
```suggestion
This action uses link:https://github.com/GoogleContainerTools/kaniko[Kaniko]to build images based upon a Dockerfile then publish the image to a Docker registry.  Kaniko builds images inside a container or kubernetes cluster. This action also reports the image and tag names to the workflow run for artifact traceability purposes.
```

**Pull request merged**